---
OMICS: "[[L4 Epigenomics]]"
---
A simple technique, that enables the detection of the methylation profile in a given sequence is the conversion of all unmethylated C to T. This conversion is catalyzed by bisulfate convertase if sodium bisulfate is provided.![](content/Attachments/Immunotek_Louella_presentation.pptx%209.png)
The conversion is specific to unmethylated C and therefore, assuming a complete conversion, all C detected after sequencing have to be methylated.
## DNA Methylation array
The Bisulfate conversion is used in DNA methylation arrays, which are pre-assembled cips, that allow for the detection of methylation. These arrays are easy to handle, cheap and highly reproducible.
The chip uses probes to capture regions of interest on the converted DNA. The most recent iterations of these chips contain close to a million different probes.

## Infinium Bead Technology
Another technique, that analyzes the methylation profile is the Infinium Bead technology. As the name suggests, this technoligy uses beads, that carry probes, to which the bisulfate converted analyte anneals. The probes contain a known sequence of a CpG cite, that we want to check for methylation. There are two approaches to do so:
1. Infinium I uses two beads for each CpG site, one corresponding to a methylated sequence and one to an unmethylated sequence. When performing a single nucleotide extension reaction with a florescence marked Nucleotide, only the probe that represents the sequence correctly (methylated vs unmethylated) produces a signal.
2. Infinium II is able to detect the weather there is a methylation by detecting the Nucleotide in the bisulfate converted sequence (where a T means unmethylated and a C methylated)![](content/Attachments/Immunotek_Louella_presentation.pptx%2010.png)

---
Created: 2024-12-03 16:39